model/model_training/trainer_sft.py

```diff
@@ -189,6 +190,7 @@ def argument_parsing(notebook=False, notebook_args=None):
     conf["wandb_entity"] = args.wandb_entity
     conf["local_rank"] = args.local_rank
     conf["deepspeed"] = args.deepspeed
+    conf["per_digit_tokens"] = args.per_digit_tokens
```

Comment:
i would move this to the config.yaml instead of args

Reply:
How about both?  Allowing command line to override it.

Reply:
this is done by default